src/package.xml

```diff
@@ -86,6 +88,7 @@
         <members>Subscription__c</members>
         <members>frField__mdt</members>
         <members>frMapping__c</members>
+        <members>Sync_Attempt__c</members>
```

Comment:
It looks like this object might be missing from the repo, and adding it here started to cause a deployment failure:
BUILD FAILED
*********** DEPLOYMENT FAILED ***********
Request ID: 0Af6R00000coIAUSA2

All Component Failures:
1.  package.xml (Sync_Attempt__c) -- Error: An object 'Sync_Attempt__c' of type CustomObject was named in package.xml, but was not found in zipped directory

*********** DEPLOYMENT FAILED ***********